main.py

```diff
@@ -184,6 +186,7 @@ async def sonidos(ctx):
 async def search(ctx, arg):
     blank_space = "\u2800"
     sounds_list = get_sound_list_filtered(arg)
+    database.register_user_interaction_async(ctx.author.global_name, "buscar")
```

Comment:
Por consistencia mejor poner todos los nombres en inglés

Reply:
Y mejor usar la ID o el nombre de usuario en vez del alias

Reply:
Analytics naming is following the !help command naming.

![image](https://github.com/Wikijito7/ECIBot/assets/18752525/6971ec6e-520b-4844-b599-0006a40fac59)